cluster/charts/xgql/values.yaml.tmpl

```diff
@@ -15,7 +15,9 @@ affinity: {}
 # tlsSecret: xgql-certs
 # args: {"--tls-key=/tls/tls.key", "--tls-cert=/tls/tls.crt"}
 
-args: {}
+args: {
+  "--cache-expiry=336h"
+}
```

Comment:
I actually haven't seen this format but based on the commented out lines above I am guessing it works! 👍🏻